core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -323,8 +326,11 @@ class GroupCoordinator(val brokerId: Int,
       responseCallback(Errors.GROUP_COORDINATOR_NOT_AVAILABLE.code)
     } else if (!isCoordinatorForGroup(groupId)) {
       responseCallback(Errors.NOT_COORDINATOR_FOR_GROUP.code)
+    } else if (isCoordinatorLoadingInProgress(groupId)) {
+      // the group is still loading, so respond just blindly
+      responseCallback(Errors.NONE.code)
```

Comment:
I originally did the special error returning, but then I changed it to just abstracting it at the server side for the same reason @hachikuji mentioned. I feel if it is mostly the case that users will not special handle this error, we'd better simply the possible return error cases in the protocol.